coverart_redirect.conf.ctmpl

```diff
@@ -0,0 +1,21 @@
+{{- define "KEY" -}}
+    {{ key (printf "docker-server-configs/CAA-Redirect/config.%s.json/%s" (env "DEPLOY_ENV") .) }}
+{{- end -}}
+
+{{if service "postgres-master"}}
+{{with index (service "postgres-master") 0}}
+[database]
+name={{template "KEY" "musicbrainz_db/name"}}
+user={{template "KEY" "musicbrainz_db/user"}}
+password={{template "KEY" "musicbrainz_db/password"}}
+host={{.Address}}
+port={{.Port}}
+{{end}}
+{{end}}
+
+[listen]
+address=0.0.0.0
+port=62080
```

Comment:
This won't run on the host network so it's better to use the default port internally (8080?).